### PR TITLE
[docs] Getting started updates

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Update buttons style on step 2 of the getting started.

## Changelog entries
```changes
section: docs
type: fix
summary: Update button style on step 2 of the getting started.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/css/components/common/_button.scss [hunk 1]
```diff
@@ -22,12 +29,16 @@
...
-    &:not(.button_disabled):hover {
+    &:not(.button_disabled):not(.button_card):hover {
         color: $color-inverse !important;
         background: $color-main;
         border: 2px solid $color-main;
         text-decoration: none !important;
     }
+    &_card:not(.button_disabled):hover {
+        transform: scale(1.1);
+        will-change: transform;
```
Comment (z9r5) [inline docs/site/css/components/common/_button.scss:40]:
```suggestion
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by ToyTox — APPROVED** [review]

**Pull request merged**